### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by danielsteenberg**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,116 @@
 <!DOCTYPE html>
```
Comment (ryannealmes) [inline app/views/layouts/application.html.erb]: Some things I would recommend here:
- Make use of partials with good naming conventions - like navigation, head or something along those lines
- Use a better templating language - slim or haml are great. Whatever you are more comfortable with.

File: app/assets/javascripts/contact_me.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/*
```
Comment (ryannealmes) [inline app/assets/javascripts/contact_me.js]: I am not too familiar with bootstrap validations, but this is probably not the best place to put it. It's generic to the whole application. It should be exposed as a library or generic javascript file. Is there no gem that you can drop into your gemfile?  Maybe it's part of the bootstrap gem already and you just need to include it in your applications.js.

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```
Comment (ryannealmes) [inline app/views/welcome/index.html.erb]: Is there a reason you have a whole html page here? Do you not want to render this with your base layout?

**Comment by ryannealmes** [comment]
Another general comment is once you have finished all your changed can you git rebase your branch and squash all the commits. Also there is no description of what you are actually trying to do for the PR. This is really frustrating and can be seen as disrespectful as well if you want someone else to look at your code. Anyway just a heads up. Great to see you working on this again.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (danielsteenberg) [inline app/views/layouts/application.html.erb]: i have sorted this out and started using Slim, took a while but will get the hang of it for sure, much more neater

File: app/assets/javascripts/contact_me.js (hunk 2 above)
Comment (danielsteenberg) [inline app/assets/javascripts/contact_me.js]: Im looking at asset pipeline, will this be the right thing to look at for this?

File: app/views/welcome/index.html.erb (hunk 3 above)
Comment (danielsteenberg) [inline app/views/welcome/index.html.erb]: I have edited it now, just has the home page content that i needed and no header or title information for the welcome page. So this page will be yielded in the application html page

File: Gemfile [hunk 4]
```diff
@@ -23,6 +23,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
```
Comment (ryannealmes) [inline Gemfile]: Please can you clear out all the comments in the gemfile.

**Comment by ryannealmes** [comment]
Is there a reason you are not using the bootstrap gem? I see you have created the bootstrap files directly in the asset pipeline.

File: app/views/shared/_header.html.slim [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    meta charset="utf-8"
+    meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="viewport" content="width=device-width, initial-scale=1"
+    meta name="description" content=""
+    meta name="author" content=""
+
+    title Asset Manager
+
+    / Bootstrap Core CSS
+    link href="css/bootstrap.min.css" rel="stylesheet"
```
Comment (ryannealmes) [inline app/views/shared/_header.html.slim]: You are already including all stylesheets and javascript files in the asset pipeline. You shouldn't need these statements here.

File: app/views/layouts/application.html.slim [hunk 6]
```diff
@@ -0,0 +1,11 @@
+= render "shared/header"
```
Comment (ryannealmes) [inline app/views/layouts/application.html.slim]: Isn't this all support to be wrapped in 

``` slim
doctype html
html lang="en"
```

File: app/views/shared/_nav.html.slim [hunk 7]
```diff
@@ -0,0 +1,70 @@
+nav class="navbar navbar-inverse navbar-fixed-top" role="navigation"
+        div class="container"
```
Comment (ryannealmes) [inline app/views/shared/_nav.html.slim]: It looks like you are using tabs for indentation. Usually we use 2 spaces.

File: app/views/shared/_nav.html.slim [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+                    span class="icon-bar"
+                    span class="icon-bar"
+                    span class="icon-bar"
+                
+                a class="navbar-brand" href="welcome/index"Asset Manager
+            
+            / Collect the nav links, forms, and other content for toggling
+            div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"
+                ul class="nav navbar-nav navbar-right"
+                    li
```
Comment (ryannealmes) [inline app/views/shared/_nav.html.slim]: A nice way to clean this up would be
`li: a href ...`

File: app/views/welcome/index.html.slim [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+          | If you interested in using this system, please click on the sign up
+          | or login button on the right
+      .col-md-4
+        a.btn.btn-lg.btn-default.btn-block href="#"  Login/Sign up
+  hr/
+  /! Footer
+
+/! /.container
+/! jQuery
+script src="js/jquery.js"
```
Comment (ryannealmes) [inline app/views/welcome/index.html.slim]: Not really sure why you have javascript here. This stuff should be done in your asset pipeline.

File: Gemfile (hunk 4 above)
Comment (danielsteenberg) [inline Gemfile]: Sorted

File: app/views/layouts/application.html.slim (hunk 6 above)
Comment (danielsteenberg) [inline app/views/layouts/application.html.slim]: Added this to the Application file

File: app/views/shared/_nav.html.slim (hunk 7 above)
Comment (danielsteenberg) [inline app/views/shared/_nav.html.slim]: will change this to two spaces to the application now

File: Gemfile (hunk 4 above)
Comment (ryannealmes) [inline Gemfile]: I still see comments in this file.

File: app/views/shared/_header.html.slim (hunk 5 above)
Comment (ryannealmes) [inline app/views/shared/_header.html.slim]: You haven't responded to this.

File: app/views/welcome/index.html.slim (hunk 9 above)
Comment (ryannealmes) [inline app/views/welcome/index.html.slim]: Same here. Not sure why you are including jquery.

File: Gemfile (hunk 4 above)
Comment (danielsteenberg) [inline Gemfile]: will push up update in a bit

File: app/views/shared/_header.html.slim (hunk 5 above)
Comment (danielsteenberg) [inline app/views/shared/_header.html.slim]: Sorted, will push up in a bit

File: app/views/welcome/index.html.slim (hunk 9 above)
Comment (danielsteenberg) [inline app/views/welcome/index.html.slim]: Sorted in the asset pipeline now.

File: app/assets/stylesheets/application.css [hunk 10]
```diff
@@ -13,3 +13,5 @@
  *= require_tree .
```
Comment (atainton) [inline app/assets/stylesheets/application.css]: this file has been duplicated, there is one scss and css, please correct

File: app/assets/stylesheets/homepage.css.scss [hunk 11]
```diff
@@ -0,0 +1,6 @@
+.header {
+	width: 100%;
```
Comment (atainton) [inline app/assets/stylesheets/homepage.css.scss:2]: i thought we deleted these files bootstrap.css and bootstrap.min.css, why are they still in the project

File: app/views/shared/_header.html.slim (hunk 5 above)
Comment (atainton) [inline app/views/shared/_header.html.slim]: why is this line still here

File: app/views/shared/_header.html.slim [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+    / Custom CSS
+    link href="css/modern-business.css" rel="stylesheet"
```
Comment (atainton) [inline app/views/shared/_header.html.slim]: why is this here

File: app/views/shared/_nav.html.slim [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+        li
+          a href="contact.html"Contact
+
+        li
+          a href="contact.html"Employees
+                    
+        li.dropdown
+          a href="#" class="dropdown-toggle" data-toggle="dropdown"
+            ul class="dropdown-menu"
+              li: a href="portfolio-1-col.html"1 Column Portfolio
```
Comment (atainton) [inline app/views/shared/_nav.html.slim:25]: looking good

File: app/assets/stylesheets/application.css (hunk 10 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/application.css]: This file is not on my side anymore, was deleted when we deleted it

File: app/assets/stylesheets/homepage.css.scss (hunk 11 above)
Comment (danielsteenberg) [inline app/assets/stylesheets/homepage.css.scss:2]: This was all deleted when we deleted it, are you looking at an older push up? Check out the latest push up requests

File: app/views/shared/_header.html.slim (hunk 12 above)
Comment (danielsteenberg) [inline app/views/shared/_header.html.slim]: Removed this links

File: app/views/shared/_nav.html.slim (hunk 13 above)
Comment (danielsteenberg) [inline app/views/shared/_nav.html.slim:25]: thanks buddy

Comment (danielsteenberg) [inline app/views/shared/_nav.html.slim:25]: thanks for the help lol

File: app/assets/javascripts/global.js [hunk 14]
```diff
@@ -0,0 +1,3 @@
+$('.carousel').carousel({
+	interval: 5000 //changes the speed
```
Comment (atainton) [inline app/assets/javascripts/global.js]: please make sure spacing is correct

**Comment by atainton** [comment]
Please fix the spacing one the one comment, then i am happy
